tests/Appsec/Mock.php

```diff
@@ -63,7 +63,7 @@ public function addEvent(array $event, $eventName)
             $this->getDbPdo()->exec(sprintf("INSERT INTO appsec_events VALUES ('%s')", json_encode($event)));
         }
 
-        public function getEvents()
```

Comment:
Changing the signature of this method not necessary on this PR but on a incoming PR(which is the one which triggered this refactor), this new signature is required. It's not changing any behaviour on this PR